syft/frameworks/torch/mpc/fss.py

```diff
@@ -41,7 +41,58 @@ def full_name(f):
 
 # number of processes
 N_CORES = 8
-MULTI_LIMIT = 10_000
+MULTI_LIMIT = 50_000
+
+
+def keygen(n_values, op):
+    """
+    Run FSS keygen in parallel to accelerate the offline part of the protocol
+
+    Args:
+        n_values (int): number of primitives to generate
+        op (str): eq or comp <=> DPF or DIF
+    """
+    if op == "eq":
+        return DPF.keygen(n_values=n_values)
+    elif op == "comp":
+        if n_values > MULTI_LIMIT:
```

Comment:
I am thinking that the code from ```op == "com"``` might be included in a separate function, but we can create a refactor ticket for it :D